src/ui/public/field_format_editor/editors/url/url.html

```diff
@@ -8,6 +8,16 @@
   </select>
 </div>
 
+<div class="checkbox" ng-if="editor.formatParams.type == 'a'">
```

Comment:
Would you mind changing this to `class="kuiCheckBox"`? This is the class provided by the [UI Framework[(https://github.com/elastic/kibana/tree/master/ui_framework).